base/libgit2/commit.jl

```diff
@@ -50,8 +50,8 @@ function commit(repo::GitRepo, msg::AbstractString;
                 refname::AbstractString=Consts.HEAD_FILE,
                 author::Signature = Signature(repo),
                 committer::Signature = Signature(repo),
-                tree_id::Oid = Oid(),
-                parent_ids::Vector{Oid}=Oid[])
+                tree_id::AbstractGitHash = GitHash(),
+                parent_ids::Vector{AbstractGitHash}=AbstractGitHash[])
```

Comment:
does this usually get passed as a vector of one or the other concretely-typed hash objects?